src/services/review/RepoCheckerService.ts

```diff
@@ -190,6 +254,11 @@ export default class RepoCheckerService {
   }
 
   remover(repo: string, repoExists: boolean) {
+    const NODE_ENV = config.get("env")
+    if (NODE_ENV === "prod") {
+      // by right this should not happen, but just in case
+      return okAsync(undefined)
+    }
```

Comment:
sorry, would it be possible to get a more complete explanation here?

Reply:
thanks for the callout, lmk if still not clear